web-components/src/components/tabs/Tabs.ts

```diff
@@ -1210,34 +1259,59 @@ export namespace Tabs {
                 : {}
             )}"
           >
-            ${repeat(
-              this.tabsFilteredAsHiddenList,
-              () => generateSimpleUniqueId("tabs"),
-              (tab) => html`
-                <md-tab
-                  slot="draggable-item"
-                  .disabled="${tab.disabled}"
-                  .selected="${tab.selected}"
-                  name="${tab.name}"
-                  id="${this.getCopyTabId(tab)}"
-                  aria-label=${tab.ariaLabel}
-                  aria-controls="${this.getAriaControlId(tab)}"
-                  @click="${() => this.handleOverlayClose()}"
-                  tabIndex="${this.tabHiddenIdPositiveTabIndex === tab.id ? 0 : -1}"
-                  ariaRole="menuitem"
-                  .newMomentum=${this.newMomentum}
-                  type=${this.type}
-                  .onlyIcon="${tab.onlyIcon}"
-                >
-                  ${unsafeHTML(tab.innerHTML)}
-                </md-tab>
-              `
-            )}
+            ${this.moreMenuListTemplate}
           </div>
         </md-menu-overlay>
       `;
     }
 
+    private get renderTabSlot() {
+      return html`<slot
+        name="tab"
+        class="${classMap({
+          "visible-tabs-slot": this.direction === "horizontal"
+        })}"
+      ></slot> `;
+    }
```

Comment:
in the case of the horizontal tabs this is added and then hidden. It is used to create new md-tabs from their properties. this should be revisited as it can cause issues